jetty-start/src/main/java/org/eclipse/jetty/start/Module.java

```diff
@@ -509,9 +521,24 @@ public String toString()
         return new HashSet<>(_provides);
     }
 
+    public Set<String> getBefore()
+    {
+        return Set.copyOf(_before);
```

Comment:
Probably would have been more efficient to return an immutable view of the Set, but as this was previous behaviour to return a copy ....